python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1362,10 +1646,29 @@ def renumber_edges_by_type(self):
 
         Returns
         -------
-        a DataFrame with the start and stop IDs for each edge type.
-        Stop is *inclusive*.
-        """
+        DataFrame
+            with the start and stop IDs for each edge type.Stop is *inclusive*.
 
+        Examples
+        --------
+        >>> import cugraph
+        >>> import cudf
+        >>> from cugraph.experimental import PropertyGraph
+        >>> pG = PropertyGraph()
+        >>> df = cudf.DataFrame(columns=["src", "dst", "edge_ids" ,"some_property"],
+        ...                              data=[(99, 22, 3, "a"),
+        ...                                    (98, 34, 5, "b"),
+        ...                                    (97, 56, 7, "c"),
+        ...                                    (96, 88, 11, "d"),
+        ...                                  ])
+        >>> pG.add_edge_data(df,
+                            type_name="etype",
+                            vertex_col_names=("src", "dst"),
+                            edge_id_col_name="edge_ids")
+        >>> pG.renumber_edges_by_type()
```

Comment:
same with this one: probably should be showing how this works with multiple types.

Reply:
second type added.